share/completions/git.fish

```diff
@@ -1731,7 +1731,7 @@ complete -f -c git -n '__fish_git_using_command format-patch' -l function-contex
 complete -f -c git -n '__fish_git_using_command format-patch' -l ext-diff -d "Allow an external diff helper to be executed"
 complete -f -c git -n '__fish_git_using_command format-patch' -l no-ext-diff -d "Disallow external diff helpers"
 complete -f -c git -n '__fish_git_using_command format-patch' -l no-textconv -d "Disallow external text conversion filters for binary files (Default)"
-complete -f -c git -n '__fish_git_using_command format-patch' -l textconv -d "Allow external text conversion filters for binary files (Resulting diff is unappliable)"
+complete -f -c git -n '__fish_git_using_command format-patch' -l textconv -d "Allow external txt conv filters for binary files (Resulting diff is unappliable)"
```

Comment:
Just "filters" or "conversion filters" should do, I think?

"Allow external filters for binary files (Resulting diff is unappliable)"